sys/shell/commands/shell_commands.c

```diff
@@ -196,6 +200,9 @@ const shell_command_t _shell_command_list[] = {
     {"ncache", "manage neighbor cache by hand", _ipv6_nc_manage },
     {"routers", "IPv6 default router list", _ipv6_nc_routers },
 #endif
+#ifdef MODULE_GNRC_IPV6_WHITELIST
+    {"whitelist", "whitelists an address for receival ('whitelist [add|del|help]')", _whitelist },
```

Comment:
Why, the thought was, that it might be expanded for IPv4 (or link-layer) some day :-)